includes/classes/blocks/class-bindings.php

```
@@ -429,6 +431,7 @@ public function build_itinerary_field( $build = '', $field = '', $count = 1 ) {
 		} else {
 			// Or set the prefix.
 			$value = $prefix . $value;
+			$value = apply_filters( 'lsx_to_itinerary_field_value', $value, $field, get_the_ID() );
 		}
```

PHPMD (2.15.0)

[error] The method build_itinerary_field uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)